src/spreadPath.js

```diff
@@ -0,0 +1,29 @@
+import { curry, pathOr, merge, dissoc } from 'ramda';
+
+/**
+ * Merges object under path with root object, removing path to object.
+ *
+ * @func spreadPath
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.19.0|v1.19.0}
+ * @category Object
+ * @sig [k] -> {a: *} -> {b: *}
+ * @param {!Object} path The path to object to merge with
+ * @param {!Object} obj The root object
```

Comment:
Let's borrow the lingo from [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator)

`The provided object`